drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -37,4 +37,16 @@ config NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE
 	  data is larger than the configured limit.  Must be a
 	  multiple of 4.  The feature is disabled when set to 0.
 
+config NORDIC_QSPI_NOR_IDLE_IN_DPD
```

Comment:
This PR is designed to follow the same logic as CONFIG_SPI_NOR_IDLE_IN_DPD for the jedec,spi-nor driver.  I can switch it to using a DTS flag if that's more desirable, just wanted to make it an easy move from using jedec,spi-nor to nordic,qspi-nor

There's some advantage to running this, even with PM_DEVICE enabled, as this will put the flash into DPD between each read/write/command, while usage of device power management would use DPD only when the whole system went into suspend, or manually triggered by the application level.

For the flash chip we're using (Winbond W25Q128JV series), the time to enter and exit DPD is small (3us each side), so it's more power efficient to shut it down between each command than at system standby or at application layer above the FCB, NVS, and flash stream drivers / interfaces.

Reply:
Thanks for explanation. I understand your rationale - they advocates this point.